src/style.css

```diff
@@ -1,3 +1,89 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --hover-color: #00acdc;
+  --menu-height: 60px;
+  --bar-height: 4px;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  background-color: lightgray;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.logo__img {
+  padding: 10px 0 10px 50px;
+  object-fit: contain;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__link {
+  display: inline-block;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+.nav__menu {
+  display: flex;
+  justify-content: flex-end;
+  list-style: none;
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+  font-size: 0;
```

Comment:
looks redundant

Reply:
I took out the 2 margin-related ones, but kept the padding-inline-start:0 and the font-size:0 to overwrite the ul default settings, otherwise, there are extra ul-specific margins and paddings, which aren't canceled by the margin:0 and padding:0 in the body